test/eth/index.spec.js

```diff
@@ -0,0 +1,40 @@
+const assert = require("assert");
+const { addGenesisTransfers } = require("../../blockchains/eth/lib/genesis_transfers")
+const Web3 = require('web3')
+const web3 = new Web3()
+
+describe('genesis transfers', function() {
```

Comment:
Nice tests!

Reply:
Can't take any credit for those - just a copy-paste.